src/workloads/basic/ReadOnlyMultiThreaded.yml

```diff
@@ -0,0 +1,91 @@
+SchemaVersion: 2018-07-01
+Owner: Product Performance
+Description: |
+  Runs a multi-threaded, read-only workload comprising of `findOne` commands that all target the 
```

Comment:
What are the main metrics to look at for this workload, can you document them here?

Reply:
Could you add some of the rationale for this test from the other tickets?

```
We will benefit from very stable benchmarks, where the primary objective is to identify regressions introduced by individual commits, and not necessarily measuring the end-to-end database performance. Such benchmarks are the building blocks for adding performance tests to the auto-reverter.
```